common/djangoapps/student/forms.py

```diff
@@ -9,7 +11,7 @@
 from django.contrib.auth.tokens import default_token_generator
 
 from django.utils.http import int_to_base36
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import ugettext_lazy as _, gettext
```

Comment:
I think a better change would be to import gettext as _, and explicitly use ugettext_lazy where necessary - I think that's clearer.

That said, I see 2 things wrong here: (1) you should be using *u*gettext (unicode gettext); (2) I'm surprised strings you defined here don't all require the lazy version. I don't think there's a user object available when this file is instantiated, thus you should probably be using ugettext_lazy everywhere in this file.